vignettes/articles/extending.Rmd

```diff
@@ -43,31 +43,42 @@ Often when passing a list from R to JavaScript it is desirable to remove any nul
 
 # Example
 
-Here is a small example which shows how you can integrate the Bing.com basemap layer [plugin](https://github.com/shramov/leaflet-plugins)
+Here is a small example which shows how you can integrate heatmap functionality using a [plugin](http://leaflet.github.io/Leaflet.heat/).
 
 ```{r, fig.height=4}
 library(leaflet)
 library(htmltools)
 library(htmlwidgets)
 
-bingPlugin <- htmlDependency(
-  "leaflet.plugins", "2.0.0",
-  src = normalizePath("./js"),
-  script = "Bing.min.js"
+# This tells htmlwidgets about our plugin name, version, and
+# where to find the script. (There's also a stylesheet argument
+# if the plugin comes with CSS files.)
```

Comment:
These comments point out that the example is rather sparse and could benefit from description. Personally, I'd prefer to have these comments appear in prose rather than in comments.

Reply:
e.g., I'm not *totally* clear on what `x` and `el` represent in the `onRender()` function - although I can see they're not used in this case.